src/introspection/port/ffi.rs

```diff
@@ -164,11 +164,12 @@ pub struct PortIntrospectionTopic {
 }
 
 impl PortIntrospectionTopic {
-    pub fn new() -> Topic<Self> {
-        TopicBuilder::<Self>::new("Introspection", "RouDi_ID", "Port")
+    pub fn new() -> InactiveSubscriber<Self> {
+        SubscriberBuilder::<Self>::new("Introspection", "RouDi_ID", "Port")
             .queue_capacity(1)
             .history_request(1)
-            .build()
+            .create_without_subscribe()
+            .expect("Create subscriber")
```

Comment:
What about "failed to create subscriber" or even better "failed to subscribe to port introspection topic"

Reply:
I think I will just update the PR with returning a `Result<InactiveSubscriber<Self>>` instead of just `InactiveSubscriber<Self>`